toolchain/personae.md

```diff
@@ -0,0 +1,56 @@
+---
+title: The Good Docs Toolchain
+---
+
+This document summarizes the personae for The Good Docs Toolchain, a collection of tools designed to help users create better technical documentation.
+
+The following are the primary types of users (i.e. personae) targeted by this toolchain. Each has a number of distinct needs, as outlined in the below sections.
+
+# Tech writer
```

Comment:
These should be an `h2` using `## Tech Writer`

Reply:
Fixed in latest update.